apps/frontend/src/components/layout/sidebar/sidebar-footer.tsx

```
@@ -40,55 +86,74 @@ export function SidebarFooter({ workspaceId, currentUser }: SidebarFooterProps)
       },
       { replace: true }
     )
-  }
+  }, [collapseOnMobile, setSearchParams])
 
   if (!currentUser) return null
 
   const avatarSrc = getAvatarUrl(workspaceId, currentUser.avatarUrl, 64)
+  const menuActions = useMemo<SidebarActionItem[]>(
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)